server.js

```diff
@@ -1,54 +1,74 @@
-var util = require('util');
-var url = require('url');
-var qs = require('querystring');
-var os = require('os')
+//  OpenShift sample Node application
+var express = require('express');
+var fs      = require('fs');
+var app     = express();
+var eps     = require('ejs');
+
+app.engine('html', require('ejs').renderFile);
+
 var port = process.env.PORT || process.env.port || process.env.OPENSHIFT_NODEJS_PORT || 8080;
-var ip = process.env.OPENSHIFT_NODEJS_IP || '0.0.0.0';
-var nodeEnv = process.env.NODE_ENV || 'unknown';
+var ip   = process.env.IP   || process.env.OPENSHIFT_NODEJS_IP || '0.0.0.0';
+var mongoURL = process.env.OPENSHIFT_MONGODB_DB_URL || process.env.MONGO_URL;
+if (mongoURL == null) {
+	var mongoPort = process.env.MONGODB_SERVICE_PORT || 27017;
+	var mongoDb = process.env.MONGODB_DATABASE || "pageviews";
```

Comment:
@mnagy Can you say more why? I like that these samples could run almost anywhere, my laptop, bluemix, etc.

The defaults help with local dev, other envs and misconfigured envs.